src/Domain/FunctionsOfTime/IntegratedFunctionOfTime.cpp

```diff
@@ -0,0 +1,143 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Domain/FunctionsOfTime/IntegratedFunctionOfTime.hpp"
+
+#include <algorithm>
+#include <iterator>
+#include <memory>
+#include <ostream>
+
+#include "DataStructures/DataVector.hpp"
+#include "Domain/FunctionsOfTime/ThreadsafeList.tpp"
+#include "Utilities/ErrorHandling/Error.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace domain::FunctionsOfTime {
+
+IntegratedFunctionOfTime::IntegratedFunctionOfTime() = default;
+IntegratedFunctionOfTime::IntegratedFunctionOfTime(IntegratedFunctionOfTime&&) =
+    default;
+IntegratedFunctionOfTime::IntegratedFunctionOfTime(
+    const IntegratedFunctionOfTime&) = default;
+auto IntegratedFunctionOfTime::operator=(IntegratedFunctionOfTime&&)
+    -> IntegratedFunctionOfTime& = default;
+auto IntegratedFunctionOfTime::operator=(const IntegratedFunctionOfTime&)
+    -> IntegratedFunctionOfTime& = default;
+IntegratedFunctionOfTime::~IntegratedFunctionOfTime() = default;
+
+IntegratedFunctionOfTime::IntegratedFunctionOfTime(
+    const double t, std::array<double, 2> initial_func_and_derivs,
+    const double expiration_time, bool rotation)
+    : deriv_info_at_update_times_(t), rotation_(rotation) {
+  deriv_info_at_update_times_.insert(t, std::move(initial_func_and_derivs),
```

Comment:
Several times: `std::array<double, 2>` is trivially copyable, so don't `std::move` it.